js/views/conversation_view.js

```diff
@@ -343,12 +348,15 @@
             var toast;
             if (extension.expired()) {
                 toast = new Whisper.ExpiredToast();
-                toast.$el.insertAfter(this.$el);
-                toast.render();
-                return;
             }
             if (this.model.isPrivate() && storage.isBlocked(this.model.id)) {
                 toast = new Whisper.BlockedToast();
+            }
+            if (!this.model.isPrivate() && this.model.get('left')) {
+                toast = new Whisper.LeftToast();
+            }
+
+            if (toast) {
```

Comment:
Might there ever be a time when multiple toasts fire? Do we care about showing multiple? If not, is the current order reasonable precedence (last one wins)?